patches/api/0009-Add-Raw-Byte-Entity-Serialization.patch

```diff
@@ -57,6 +59,5 @@ index ae1ebc5f8a61165d95184e2a12a270825ed1fd2d..83eddbdabddcf4db726cb210de12c679
 +     * @return Whether the entity was successfully spawned.
 +     */
 +    boolean spawnAt(@NotNull Location location, @NotNull org.bukkit.event.entity.CreatureSpawnEvent.SpawnReason spawnReason);
-     // Paper end
- 
-     // Papercut start
+     // Papercut end
```

Comment:
Is the start of this wrapper in bundled with another patch?

Reply:
It's bundled with the raw methods patch, but it shouldn't add the end here, I'll fix that later.